File: 5° semestre/Programação para dispositivos moveis II/aula-25-09-26/atividades pré prova/2 - Salario/projeto2/app/src/main/java/com/example/salario/MainActivity.java
package com.example.salario;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.RadioButton;
import android.widget.TextView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    private EditText textSalario;
    private RadioButton rb1, rb2, rb3;
    private Button btnResultado;
    private TextView textResultado;
    private double total;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);

        textSalario = findViewById(R.id.textSalario);
        rb1 = findViewById(R.id.rb1);
        rb2 = findViewById(R.id.rb2);
        rb3 = findViewById(R.id.rb3);
        btnResultado = findViewById(R.id.btnResultado);
        textResultado = findViewById(R.id.textResultado);

        btnResultado.setOnClickListener(v -> {
            String salarioStr = textSalario.getText().toString();

            if (salarioStr.isEmpty()) {
                textResultado.setText("Digite o salário!");
                return;
            }

            double salario = Double.parseDouble(salarioStr);

            if (rb1.isChecked()) {
                total = salario * 1.4; // 40% de aumento
            } else if (rb2.isChecked()) {
                total = salario * 1.45; // 45% de aumento
            } else if (rb3.isChecked()) {
                total = salario * 1.5; // 50% de aumento
            }

            textResultado.setText("R$ " + String.format("%.2f", total));
        });
    }
}

File: 3° semestre/Programação-dispositivos-moveis/aula-24-10-03/calculadoraAposentadoria/MainActivity.kt
package com.example.myapplication

import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.Button
import android.widget.EditText
import android.widget.Spinner
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val spn_Sexo = findViewById<Spinner>(R.id.spn_Sexo)
        val txt_Idade = findViewById<EditText>(R.id.txt_Idade)  // Campo de idade agora é EditText
        val btn_Calcular = findViewById<Button>(R.id.btn_Calcular)
        val txt_Resultado = findViewById<TextView>(R.id.txt_Resultado)

        spn_Sexo.adapter = ArrayAdapter<String>(this, android.R.layout.simple_spinner_dropdown_item,
            listOf("Masculino", "Feminino"))

        // btn_Calcular_onclick
        btn_Calcular.setOnClickListener {
            val sexo = spn_Sexo.selectedItem as String
            val idadeStr = txt_Idade.text.toString()

            if (idadeStr.isEmpty()) {
                txt_Idade.error = "Campo obrigatório"
                return@setOnClickListener  // Sai da função se o campo estiver vazio
            }

            val idade = idadeStr.toInt()
            var resultado = 0

            if (sexo == "Masculino") {
                resultado = 65 - idade
            } else {
                resultado = 62 - idade
            }

            txt_Resultado.text = "Faltam $resultado anos para você se aposentar"
        }
    }
}
